### PR TITLE
Image promotion for cluster-api-helm v0.1.0-alpha.1

### PR DESCRIPTION
**Pull request by Jont828**
Image promotion for cluster-api-helm v0.1.0-alpha.1
This is an automated PR generated from `kpromo`
```
kpromo pr --fork Jont828 --project cluster-api-helm --reviewers "@CecileRobertMichon @enxebre @fabriziopandini @sbueringer @vincepri" --tag v0.1.0-alpha.1 --image cluster-api-helm-controller
```

/hold
cc: @CecileRobertMichon @enxebre @fabriziopandini @sbueringer @vincepri

### CODE REVIEW

**Comment by sbueringer** [comment]
@Jont828 I think it would be better to only mention folks in the generated PR description that are maintainers of the helm provider and have the corresponding approval rights for this file/folder.

File: k8s.gcr.io/images/k8s-staging-cluster-api-helm/images.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-# No images yet
```
Comment (CecileRobertMichon) [inline k8s.gcr.io/images/k8s-staging-cluster-api-helm/images.yaml:1]: /hold

this isn't actually adding any images

**Comment by BenTheElder** [comment]
Hey, you'll have to move this to registry.k8s.io/, sorry we're working on https://github.com/kubernetes/enhancements/issues/3720 ref https://github.com/kubernetes/test-infra/pull/29069

**Comment by BenTheElder** [comment]
/close
[attempting to force the bot to process this, will reopen]

**Comment by sbueringer** [comment]
@cpanato is my understanding correct that for core CAPI we can simply use the latest kpromo and we're good? (asking because of the upcoming RC release next week)

**Comment by cpanato** [comment]
> @cpanato is my understanding correct that for core CAPI we can simply use the latest kpromo and we're good? (asking because of the upcoming RC release next week)

yes, please use the latest kpromo, we will send out a communication soon

**Pull request closed without merging**